server/portal/settings/settings_default.py

```diff
@@ -49,22 +47,16 @@
 # DJANGO APP: WORKSPACE
 ########################
 
-_PORTAL_APPS_METADATA_NAMES = ["frontera_apps", "frontera_apps_dev"]
+_PORTAL_APPS_METADATA_NAMES = []
```

Comment:
you could drop this right?  and the import-apps?  

as once me move to v3, there will be no apps in metadata to import, right?

Reply:
I'm going to refactor this to use a field in `notes` to import apps